gmx_top_prep.xml

```diff
@@ -8,38 +8,31 @@
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
 
-        #if $tgz.is_of_type("tgz"):
-            #set $tgz_input = 'charmm-gui.tgz'
-            ln -s '$tgz' '$tgz_input' &&
-        #end if
-
-        tar -zxvf $tgz_input `tar -tf charmm-gui.tgz | grep -E topol.top` &&
```

Comment:
Aswe know this is not really a tgz use the correct flag
`tar -xvf charmm-gui.tgz `tar -tf charmm-gui.tgz | grep -E topol.top` &&`